### ISSUE TITLE
Owncloud supported versions

### ISSUE THREAD

**Issue by sirio81** [owncloud/search_lucene#122]
Hi all, looking at [this page](https://apps.owncloud.com/content/show.php?content=168709&forumpage=1) it seems owncloud 8.2.X and 9.X is not supported by search_lucene.
Is that correct?

**Comment by strugee** [comment]
Yes. Glance through the issue tracker and you'll see issues about newer version support.
